### PR TITLE
chore: update .gitignore

### PR DESCRIPTION
**Pull request by kptkin**
Description
-----------

What does the PR do? Include a concise description of the PR contents.

- [x] I updated CHANGELOG.md, or it's not applicable

Testing
-------
How was this PR tested?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,49 +1,46 @@
...
-apple_stats/.build/
-
-__pycache__/
-htmlcov/
 .mypy_cache/
-.pytest_cache/
-.hypothesis/
-.circle_cache/
+
 .yea_cache/
```
Comment (dmitryduev) [inline .gitignore:38]: afuera?

File: .gitignore [hunk 2]
```diff
@@ -1,49 +1,46 @@
...
-tests/unit_tests_old/logs/*
-!tests/unit_tests_old/logs/cleanup.sh
-pip-wheel-metadata
-.vscode/.ropeproject
-xcuserdata/
-cover-results/
-wandb/proto_check/
-*.log
-.idea/
 .yea-requirements.txt
```
Comment (dmitryduev) [inline .gitignore:39]: afuera?

**Review by dmitryduev — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (kptkin) [reply .gitignore:39]: yes, i have a pr that removed left over yea stuff

Comment (kptkin) [reply .gitignore:39]: #8384

**Pull request merged**